tiledb/cf/creator.py

```diff
@@ -646,6 +646,10 @@ def __init__(
         self.offsets_filters = offsets_filters
         self.allows_duplicates = allows_duplicates
         self.sparse = sparse
+        self.__post_init__()
+
+    def __post_init__(self):
```

Comment:
`__post_init__` is a special method for dataclasses and it's called automatically. Did `ArrayCreator` use to be a dataclass before? Looks kind of odd as is.